ci/fih_test_docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 Arm Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:focal
+
+# get dependencies for retrieving and building TF-M with MCUBoot, and QEMU.
+RUN apt-get update && \
+    DEBIAN_FRONTEND="noninteractive" \
+    apt-get install -y \
+    cmake \
+    curl \
+    git \
+    libncurses5 \
+    python3 \
+    python3-pip \
+    qemu-system-arm
+
+RUN \
+    # installing python packages
+    python3 -m pip install \
+        cbor==1.0.0 \
+        cryptography==2.7 \
+        imgtool==1.6.0 \
+        Jinja2==2.10 \
+        PyYAML==3.12 \
+        pyasn1==0.1.9 &&\
+    # Install toolchain
+    mkdir -p /root/bin/gcc-7-2018-q2-update &&\
+    curl --create-dirs -fsSLo /tmp/gcc-arm-none-eabi-linux.tar.bz2 'https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2?revision=bc2c96c0-14b5-4bb4-9f18-bceb4050fee7?product=GNU%20Arm%20Embedded%20Toolchain%20Downloads,64-bit,,Linux,7-2018-q2-update' &&\
```

Comment:
Would it be a problem to just use `gcc-arm-none-eabi` and its companion packages directly from Ubuntu repositories?